### PR TITLE
Update tests from makefile to metadata (03)

### PR DESCRIPTION
**Pull request by idorax**
o vm/hugepage/libhugetlbfs
    o vm/kvm-self-tests
    o vm/kvm-unit-tests

### CODE REVIEW

File: vm/kvm-self-tests/metadata [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+name=vm/kvm-self-tests
+owner=Luiz Capitulino <lcapitulino@redhat.com>
+description=KVM self tests
+license=GPLv3
+confidential=no
+destructive=no
+
+[restraint]
+entry_point=bash ./runtest.sh
+dependencies=bc,gcc,make,patch,rsync
```
Comment (emanchado) [inline vm/kvm-self-tests/metadata:11]: How does this work with dependencies like `python2-lxml`? Don't they need to be specified somehow?

**Review by emanchado — COMMENTED** [review]
Looks good, I think? But I'm unsure about some of the dependencies.

File: vm/kvm-self-tests/metadata (hunk 1 above)
Comment (idorax) [reply vm/kvm-self-tests/metadata:11]: Hi @emanchado, please ignore `python2-lxml` and `python3-lxml` as we don't run test case manually now. Those two packages was added by me to Makefile as I once run the test case manually to save time.

Comment (emanchado) [reply vm/kvm-self-tests/metadata:11]: Ah, cool, thanks. Is this still WIP, or can I have a final look and merge if I don't have any other questions?

Comment (idorax) [reply vm/kvm-self-tests/metadata:11]: Please wait for a while as I'm still testing/debugging the changes for `vm/hugepage/libhugetlbfs` after removing its Makefile.

**Comment by idorax** [comment]
@cki-bot, please test!

**Comment by idorax** [comment]
@cki-bot, please test again!

**Comment by idorax** [comment]
CKI pipeline summary:
* http://pastebin.test.redhat.com/870890

Tests are passed on both RHEL7 and RHEL8, and those test failures of upstream are not related to current changes.

**Comment by ftjahjad** [comment]
Look Good to me,
@rasibley , could you review it :-)

**Comment by rasibley** [comment]
Changes look good to me but Libhugetlbfs is now failing for both rhel7/rhel8 where it didn't fail before, can you please investigate ? Also cc @mcondotta as he also has a pending PR to update the kvm tests. We are working on updating the tests in batches to convert the Makefiles to metadata files to remove legacy RHTS support.

**Comment by mh21** [comment]
The project is being moved to gitlab.com/cki-project/kernel-tests. Please reopen your merge request there :-)

**Pull request closed without merging**